corporate/lib/registration.py

```diff
@@ -60,9 +60,9 @@ def send_user_unable_to_signup_group_direct_message_to_admins(
     realm: Realm, user_email: str
 ) -> None:
     message = _(
-        "A new member ({email}) was unable to join your organization because all Zulip licenses "
-        "are in use. Please [increase the number of licenses]({billing_page_link}) or "
-        "[deactivate inactive users]({deactivate_user_help_page_link}) to allow new members to join."
+        "A new user ({email}) was unable to join your organization. Please increase the "
+        "[number of licenses for the current and/or next billing periods]({billing_page_link}) or "
+        "[deactivate inactive users]({deactivate_user_help_page_link}) to allow new users to join."
```

Comment:
Since this can be sent if there aren't enough licenses for a guest to join, I changed "member" to "user". Also, as a user might not be able to join due to the renewal licenses, all current purchased licenses might not be in use, to adjusted the text to cover both cases.

Reply:
The group DM text has been updated to a revised version from @alya.

> A new user ({email}) was unable to join because your organization does not have enough Zulip licenses. To allow new users to join, make sure that the number of licenses for the current and next billing period is greater than the current number of users.